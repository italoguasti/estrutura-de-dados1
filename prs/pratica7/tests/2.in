10
jesus 25 1 0000
pedro 19 10 1930
valentina 1 3 1945
siva 3 11 1968
maria 17 5 1972
jose 1 5 1990
daiane 24 12 1993
guilherme 9 3 1998
paulo 6 4 1998
giovana 8 3 2015