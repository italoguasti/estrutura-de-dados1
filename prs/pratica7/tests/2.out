Analise.....: Comp | Movi
- Bubble....: 0009 | 0000
- Selection.: 0045 | 0000
- Insertion.: 0009 | 0009
